### PR TITLE
Feature/timeline

### PR DESCRIPTION
**Pull request by ashoffmann90**
### Description
[Updated styling for Timeline view] (https://trello.com/c/BKQltFs5)
- [x] Ran locally and passed all tests

### CODE REVIEW

File: src/components/pages/Incidents/Timeline/timeline.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+const { RangePicker } = DatePicker;
+
+const TimelineLabel = props => {
+  const [endDate, setEndDate] = useState('');
+  const [startDate, setStartDate] = useState('');
+  let mode = 'left';
+  const incidents = props.incidents;
+  const [filteredIncidents, setFilteredIncidents] = useState(incidents);
+  console.log(filteredIncidents);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Timeline/timeline.js]: Remove this console.log please

File: src/components/pages/Incidents/Timeline/timeline.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  };
+
+  const setStartValue = (value, dateString) => {
+    setStartDate(dateString);
+    console.log('value', dateString);
+  };
+
+  const setEndValue = (value, dateString) => {
+    setEndDate(dateString);
+    console.log('value', dateString);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Timeline/timeline.js]: Remove please

File: src/components/pages/Incidents/Timeline/timeline.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            <LocalPopOver incident={incident} marker={false} text={true} />
+          </Timeline.Item>
+        );
+      });
+    }
+  };
+
+  const setStartValue = (value, dateString) => {
+    setStartDate(dateString);
+    console.log('value', dateString);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Timeline/timeline.js]: remove please

File: src/components/pages/Incidents/Timeline/timeline.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  const setEndValue = (value, dateString) => {
+    setEndDate(dateString);
+    console.log('value', dateString);
+  };
+
+  const newPickDate = () => {
+    return (
+      <Space direction="vertical">
+        <DatePicker onChange={setStartValue} />
+        <DatePicker onChange={setEndValue} />
```
Comment (BrannanC) [inline src/components/pages/Incidents/Timeline/timeline.js:53]: I like this formatting a lot better, much easier to read without having the function written inline

File: src/components/pages/ViewChange/viewchange.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  );
+
+  return (
+    <>
+      <div id="viewchangeparent">
+        <Dropdown overlay={menu} placement="topLeft">
+          <Button
+            style={{
+              backgroundColor: 'lightgrey',
+              position: 'fixed',
```
Comment (BrannanC) [inline src/components/pages/ViewChange/viewchange.js:27]: Do these styles need to be inline for any reason?

**Review by cwilkison — APPROVED** [review]
great move changing the date picker to a more usable component for mobile

**Pull request merged**